src/content/admin/registry-credentials/registry-credentials.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Registry Credentials
+description: Manage and configure your registry credentials
+sidebar_label: Registry Credentials
+id: registry-credentials
+---
+
+import Image from "@theme/Image";
+
+In the Admin Dashboard, you can set up private registry credentials for your Okteto instance. These credentials are automatically used by Okteto for various developer operations like building and deploying, so developers don't need direct credential access. Once set, all developers can access the registries through Okteto without additional steps. Additionally, you have the option to manage these credentials via Kubernetes CRDs provided by Okteto.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/registry_credentials_view.png").default}
+    alt="Registry credentials view"
+    width="900"
+  />
+</p>
+
+When the `daemonset.configurePrivateRegistriesInNodes.enabled` setting is active, Okteto automatically applies registry credentials to nodes, allowing cluster-wide access to these registries.
```

Comment:
@maroshii @codyjlandstrom could we document the new pull secret operator option and its advantages?

Reply:
I'll take care of the other pull secrets docs. I would merge this as-is and I'll update in the next PR